### PR TITLE
feat: default theme for button

### PR DESCRIPTION
**Pull request by enguerran**
I believe we need to provide default value for __every__ required props.

And I believe we need to set which props are required using [PropTypes](https://reactjs.org/docs/typechecking-with-proptypes.html).

(I notice we speak about _default value_ in #190 too)

### CODE REVIEW

**Comment by enguerran** [comment]
I believe this recommandation must be added everywhere. I will update this very PR with proptypes.

File: react/Button/index.jsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import React from 'react'
 import cx from 'classnames'
+import PropTypes from 'prop-types'
 import styles from './styles'
 
-export default (props) => {
-  const { theme = 'regular', busy, className, children, onClick } = props
+const Button = (props) => {
+  const { theme = 'regular', busy = false, className = '', children, onClick = () => { console.log('button was clicked') } } = props
```
Comment (GoOz) [inline react/Button/index.jsx]: You forgot to remove the `console.log`. :)

Comment (ptbrowne) [reply react/Button/index.jsx]: No default for onClick please :)

Comment (enguerran) [reply react/Button/index.jsx]: No, I don't. That is a default value for the `onClick` function.

Comment (GoOz) [reply react/Button/index.jsx]: Oh I get it, sorry. I let you two decide what's best then.

**Comment by enguerran** [comment]
Ok, let me know if I get suggestions (approval or not) by code owners.

**Comment by ptbrowne** [comment]
Thanks @enguerran  !

**Pull request merged**